src/main/kotlin/no/nav/hjelpemidler/delbestilling/roller/RunOnStart.kt

```diff
@@ -0,0 +1,53 @@
+package no.nav.hjelpemidler.delbestilling.roller
+
+import mu.KotlinLogging
+import no.nav.hjelpemidler.delbestilling.Database
+import no.nav.hjelpemidler.delbestilling.delbestilling.DelbestillingRepository
+import no.nav.hjelpemidler.delbestilling.hjelpemidler.HjelpemiddelDeler
+
+private val logg = KotlinLogging.logger {}
+
+class RunOnStart(
+    private val delbestillingRepository: DelbestillingRepository = DelbestillingRepository(Database.migratedDataSource),
+) {
+    suspend fun importNavn() {
+        delbestillingRepository.withTransaction { tx ->
+            val delbestillinger = delbestillingRepository.hentDelbestillinger(tx)
+
+            var antallOppdaterteRader = 0
+            delbestillinger.forEach { lagretDelbestilling ->
+
+                val navnHovedprodukt =
+                    HjelpemiddelDeler.hentHjelpemiddelMedDeler(lagretDelbestilling.delbestilling.hmsnr)?.navn
+
+                if (navnHovedprodukt == null) {
+                    logg.info { "Klarte ikke å finne navn for hmsnr: ${lagretDelbestilling.delbestilling.hmsnr} (saksnummer: ${lagretDelbestilling.saksnummer}), hoppper over..." }
+                    return@forEach
+                }
+
+                if (lagretDelbestilling.delbestilling.navn != null) {
+                    logg.info { "lagretBestilling med saksnummer ${lagretDelbestilling.saksnummer} har allerede navn=${lagretDelbestilling.delbestilling.navn}, hopper over..." }
+                    return@forEach
+                }
+
+                delbestillingRepository.withTransaction { tx ->
```

Comment:
Har kanskje ikkje noko å sei for ein slik eingangsjobb. Men her trur eg transaksjonen på linje 33 kan fjernast, og så berre bruke transaksjonen frå linje 14 i staden (på linje 39).